src/utils/backend/classroom/classroom.ts

```diff
@@ -409,4 +409,3 @@ export async function getClassStudentList(
     error: null,
   };
 }
-
```

Comment:
trailing whitespace empty line

Reply:
You had 2 line breaks here. Convention (and literally every other file in this project) says there should be one. I just corrected them.